recipes/compilers/meta.yaml

```diff
@@ -0,0 +1,217 @@
+{% set c_compiler = compiler('c').split('_')[0] %}
+{% set cxx_compiler = compiler('cxx').split('_')[0] %}
+{% set fortran_compiler = compiler('fortran').split('_')[0] %}
+
+{% set commands = {
+  'binutils': {
+    'linux': ['ar', 'as', 'ld', 'nm', 'size', 'strings', 'strip',
+              'addr2line', 'c++filt', 'dwp', 'elfedit', 'gprof', 'ld.bfd',
+              'ld.gold', 'objcopy', 'objdump', 'ranlib', 'readelf'],
+    'osx': ['ar', 'as', 'ld', 'nm', 'size', 'strings', 'strip',
+            'checksyms', 'codesign_allocate', 'indr', 'install_name_tool',
+            'libtool', 'lipo', 'nmedit', 'otool', 'pagestuff', 'ranlib',
+            'redo_prebinding', 'seg_addr_table', 'seg_hack', 'segedit'],
+  },
+  'c-compiler': {
+    'gcc': ['cc', 'ct-ng.config', 'cpp', 'gcc', 'gcc-ar', 'gcc-nm',
+              'gcc-ranlib', 'gcov', 'gcov-dump', 'gcov-tool'],
+    'clang': ['clang'],
+    'vs2008': [],
+    'vs2010': [],
+    'vs2015': [],
+  }[c_compiler],
+  'cxx-compiler': {
+    'gxx': ['g++', 'c++'],
+    'clangxx': ['clang++'],
+    'vs2008': [],
+    'vs2010': [],
+    'vs2015': [],
+  }[cxx_compiler],
+  'fortran-compiler': {
+    'gfortran': ['gfortran'],
+  }[fortran_compiler],
+} %}
+
+
+package:
+  name: compilers
+  version: 1.0.0
+
+build:
+  number: 0
+
+outputs:
+  - name: binutils
+    requirements:
+      host:
+        - binutils_linux-64  # [linux]
+        - cctools  # [osx]
+        - ld64  # [osx]
+      run:
+        - binutils_linux-64  # [linux]
+        - cctools  # [osx]
+        - ld64  # [osx]
+    script: build-symlinks.sh
```

Comment:
I think this needs a `# [unix]` selector.

Reply:
Yeah that fixed it, for some reason Appveyor didn't show up in the checks for the old PR.